Pipelines/azure-pipeline-release.yaml

```diff
@@ -4,8 +4,13 @@
 # Builds Microsoft.SpatialAudio.Unity binaries and packages
 
 trigger:
-- master
-- develop
+  branches:
+    include:
+    - master
+    - develop
+  paths:
+    include:
+      - Source/*

```

Comment:
What about changes under Tools and Pipelines?

Reply:
Yeah, wasn't quite sure if we want auto triggered builds for those paths but I can see it both ways. I'll add them...

---
In reply to: [356844330](https://github.com/microsoft/spatialaudio-unity/pull/9#discussion_r356844330) [](ancestors = 356844330)

Reply:
I also missed these changes for the validation pipeline, so adding those as well.

---
In reply to: [356846326](https://github.com/microsoft/spatialaudio-unity/pull/9#discussion_r356846326) [](ancestors = 356846326,356844330)